Spec file fails to build on CentOS 8/9 stream

**Describe the bug**
I tried to run `make rpm` on Cent OS 8 stream and the build failed.

**GEOPM version**
9f715aea16de1cad848ec2dbd773b0a143fc0a9a

**Expected behavior**
Clean build of RPM on CentOS

**Actual behavior**
The rpmbuild step fails.

**Error messages produced**

```
...
Processing files: geopm-service-debugsource-2.0.0+dev464g9f715aea-1.x86_64
error: Empty %files file /root/rpmbuild/BUILD/geopm-service-2.0.0+dev464g9f715aea/debugsourcefiles.list


RPM build errors:
    Macro expanded in comment on line 13: %{_libdir} and %{_includedir}.

    Empty %files file /root/rpmbuild/BUILD/geopm-service-2.0.0+dev464g9f715aea/debugsourcefiles.list
make: *** [Makefile:3971: rpm] Error 1
```

--- a/service/geopm-service.spec.in
+++ b/service/geopm-service.spec.in
@@ -10,7 +10,7 @@
 #
 # 2. rpmbuild --define 'enable_level_zero 1' ...
 #    Enable level-zero with packages installed into standard
-#    locations: %{_libdir} and %{_includedir}.
+#    locations: libdir and includedir.
 #
 # 3. rpmbuild --define 'with_level_zero <LEVEL_ZERO_PREFIX>' ...
 #    Enable level-zero with packages installed into a
@@ -58,6 +58,8 @@ BuildRequires: python-rpm-macros
 %else
 %define docdir %{_defaultdocdir}/geopm-service-%{version}
 %endif
+
+%global debug_package %{nil}
 
 %define python_bin %{__python3}
 
@@ -172,12 +174,13 @@ CFLAGS= CXXFLAGS= CC=gcc CXX=g++ \
 %{__make} DESTDIR=%{buildroot} install
 rm -f $(find %{buildroot}/%{_libdir} -name '*.a'; \
         find %{buildroot}/%{_libdir} -name '*.la')
+
 install -Dp -m644 geopm.service %{buildroot}%{_unitdir}/geopm.service
 install -Dp -m644 io.github.geopm.conf %{buildroot}%{_datadir}/dbus-1/system.d/io.github.geopm.conf
 install -Dp -m644 io.github.geopm.xml %{buildroot}%{_datadir}/dbus-1/interfaces/io.github.geopm.xml
 mkdir -p %{buildroot}%{_sysconfdir}/geopm-service
 mkdir -p %{buildroot}%{_sbindir}
-ln -s /usr/sbin/service %{buildroot}%{_sbindir}/rcgeopm
+ln -s -r service %{buildroot}%{_sbindir}/rcgeopm
 %if 0%{?suse_version}
 %python_expand %fdupes %{buildroot}%{python3_sitelib}
 %endif

